@AllanOXDi It seems there are some [failing tests](https://github.com/learningequality/kolibri/runs/8155964554?check_suite_focus=true#step:6:1064). 
```
Traceback (most recent call last):
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/rest_framework/viewsets.py", line 116, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/rest_framework/views.py", line 492, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/runner/work/kolibri/kolibri/kolibri/core/api.py", line 299, in list
    queryset = self.filter_queryset(self.get_queryset())
  File "/home/runner/work/kolibri/kolibri/kolibri/core/content/api.py", line 1398, in get_queryset
    content_id=OuterRef("content_id"), user=user
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/db/models/query.py", line 787, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/db/models/query.py", line 805, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/db/models/sql/query.py", line 1250, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/db/models/sql/query.py", line 1276, in _add_q
    allow_joins=allow_joins, split_subq=split_subq,
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/db/models/sql/query.py", line 1179, in build_filter
    self.check_related_objects(field, value, opts)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/db/models/sql/query.py", line 1076, in check_related_objects
    self.check_query_object_type(value, opts, field)
  File "/home/runner/work/kolibri/kolibri/.tox/py3.5/lib/python3.5/site-packages/django/db/models/sql/query.py", line [1054](https://github.com/learningequality/kolibri/runs/8155964554?check_suite_focus=true#step:6:1055), in check_query_object_type
    (value, opts.object_name))
ValueError: Cannot query "AnonymousUser": Must be "FacilityUser" instance.
```

--- a/AUTHORS.md
+++ b/AUTHORS.md
@@ -5,7 +5,7 @@ Kolibri is copyrighted by Learning Equality under the MIT License.
 If you have contributed to Kolibri, feel free to add your name and Github account to this list:
 
 | Name | Github user |
-|------|-------------|
+|--|-------------|
 | Eli Dai | 66eli77 |
 | Adam Stasiw | AdamStasiw |
 | Akshay Mahajan | akshaymahajans |
@@ -84,3 +84,4 @@ If you have contributed to Kolibri, feel free to add your name and Github accoun
 | Carol Willing | willingc |
 | Yash Jipkate | YashJipkate |
 | Yixuan Liu | yil039 |
+| Allan Otodi | AllanOXDi |

--- a/kolibri/core/content/api.py
+++ b/kolibri/core/content/api.py
@@ -1391,7 +1391,11 @@ class UserContentNodeViewset(BaseContentNodeMixin, BaseValuesViewset, ListModelM
 
     def get_queryset(self):
         user = self.request.user
+
         queryset = models.ContentNode.objects.filter(available=True)
+        if not user.is_facility_user:
+            user = None
+
         queryset = queryset.annotate(
             last_interacted=Subquery(
                 ContentSummaryLog.objects.filter(
@@ -1430,7 +1434,11 @@ class ContentNodeProgressViewset(
 
     def get_queryset(self):
         user = self.request.user
+
         queryset = models.ContentNode.objects.filter(available=True)
+        if not user.is_facility_user:
+            user = None
+
         queryset = queryset.annotate(
             last_interacted=Subquery(
                 ContentSummaryLog.objects.filter(

